IPython/core/magics/execution.py

```diff
@@ -71,23 +72,27 @@ class TimeitResult(object):
 
     """
 
-    def __init__(self, loops, repeat, best, worst, all_runs, compile_time, precision):
+    def __init__(self, loops, repeat, average, stdev, all_runs, compile_time, precision):
         self.loops = loops
         self.repeat = repeat
-        self.best = best
-        self.worst = worst
+        self.average = average
+        self.stdev = stdev
```

Comment:
I was thinking about that approach but I think that because those numbers are not going to change and because the printed message to stdout is much more informative with the proper statistics I think that the current way is good enough. in cases you think something would suit better I more than happy to adapt the code.